File: backend/app/Domain/Doctor/Http/Requests/LoginRequest.php
<?php

namespace App\Domain\Doctor\Http\Requests;

use Illuminate\Foundation\Http\FormRequest;

class LoginRequest extends FormRequest
{
    public function authorize()
    {
        return true;
    }

    public function rules()
    {
        return [
            'identifier' => 'required|string',
            'password' => 'required|string',
        ];
    }
}

File: mobile_app/respilink_mobile/lib/features/auth/data/models/requests/login_request.dart
class LoginRequest {
  final String itsNumber;
  final String password;
  final String fcmToken;

  LoginRequest({
    required this.itsNumber,
    required this.password,
    required this.fcmToken,
  });

  Map<String, dynamic> toJson() {
    return {
      'identifier': itsNumber,
      'password': password,
      //'fcm_token': fcmToken,
    };
  }
}